NewProject/NewProject/ViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ViewController.swift
```

Comment:
please remove unused files from the code base

Reply:
sure